README.md

```diff
@@ -69,7 +39,6 @@ The following examples assume you have `$KUBECONFIG` set.
   * `cluster.k8s.io/cluster-name=<cluster name>`
```

Comment:
We should also include the `Version` field on the MachineSpec is filled out otherwise we get:

```
root@ip-172-31-60-121:~/cluster-api-upgrade-tool# ./cluster-api-upgrade-tool --cluster-name my-cluster --cluster-namespace default --kubeconfig /etc/kubernetes/admin.conf --kubernetes-version v1.16.2 --upgrade-id 1
INFO[0000] Retrieving cluster from management cluster    cluster-name=my-cluster cluster-namespace=default
INFO[0000] Creating target kubernetes client            
INFO[0000] Rerun with `--upgrade-id=1` if this upgrade fails midway and you want to retry 
INFO[0000] Listing machines                              labelSelector="map[cluster.x-k8s.io/cluster-name:my-cluster cluster.x-k8s.io/control-plane:true]"
Error: error determining current control plane versions: nil control plane version for machine default/controlplane-0
nil control plane version for machine default/controlplane-0
github.com/vmware/cluster-api-upgrade-tool/pkg/upgrade.(*ControlPlaneUpgrader).minMaxControlPlaneVersions
	/home/ubuntu/cluster-api-upgrade-tool/pkg/upgrade/control_plane.go:231
github.com/vmware/cluster-api-upgrade-tool/pkg/upgrade.(*ControlPlaneUpgrader).Upgrade
	/home/ubuntu/cluster-api-upgrade-tool/pkg/upgrade/control_plane.go:151
main.main.func1
	/home/ubuntu/cluster-api-upgrade-tool/main.go:36
github.com/spf13/cobra.(*Command).execute
	/home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762
github.com/spf13/cobra.(*Command).ExecuteC
	/home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852
github.com/spf13/cobra.(*Command).Execute
	/home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main
	/home/ubuntu/cluster-api-upgrade-tool/main.go:89
runtime.main
	/home/ubuntu/.gimme/versions/go1.12.12.linux.amd64/src/runtime/proc.go:200
runtime.goexit
	/home/ubuntu/.gimme/versions/go1.12.12.linux.amd64/src/runtime/asm_amd64.s:1337
```

Reply:
+1

Reply:
Separate issue though 😄

Reply:
yep, agreed. https://github.com/vmware/cluster-api-upgrade-tool/issues/109